string[] FrontLeft   
string[] FrontRight   
string[] RearLeft     
string[] RearRight    
string[] Trunk        